lib/beaker-http/http.rb

```diff
@@ -3,22 +3,26 @@ module Http
 
     # Beaker Http objects are essentially wrappers for a connection object that
     # utilizes a Beaker::Host object for easier setup during testing.
-    class Http
+    class Connection
       include Beaker::Http::Helpers
       extend Forwardable
 
-      attr_accessor :connection, :host
+      attr_accessor :connection
+      attr_reader :host
 
       def initialize(host)
```

Comment:
good point.  this needs to be better in beaker.  all methods that take a host should be able to take a hostname also.